### PR TITLE
update officeholder email, address and role

### PR DESCRIPTION
**Pull request by cballantyne**
Can only be called by officeholder for organisation.

### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] If this change fixes a bug (or a performance problem), is a regression
  test (or a benchmark) included?
- [ ] Is a documentation update included (if this change modifies
  existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### CODE REVIEW

**Comment by dpnova** [comment]
Looks like there's some inconsistency of the capitalisation of the H in officeholder.

**Comment by cballantyne** [comment]
Just pushed fix for inconsistencies of H capitalisation. We will move forward with officeholder instead of officeHolder. Will make another ticket to address rest of inconsistencies that have crept in.

File: web/web/tests/test_graphene.py [hunk 1]
```diff
@@ -964,3 +964,124 @@ def test_ShareholderUtils_is_user_linked_to_shareholder_true(self):
                 user.id
             )
         )
+
+    def test_UpdateOfficeHolder_mutate_success(self):
```
Comment (ejjick) [inline web/web/tests/test_graphene.py]: casing

File: web/web/tests/test_graphene.py [hunk 2]
```diff
@@ -964,3 +964,124 @@ def test_ShareholderUtils_is_user_linked_to_shareholder_true(self):
...
+            'hello@example.com',
+            officeholder_1_db.email
+        )
+        self.assertEqual(
+            'CEO',
+            officeholder_1_db.role
+        )
+
+    # User is not an officeholder so mutate should return error.
+    def test_UpdateOfficeHolder_mutate_no_officeholder_error(self):
```
Comment (ejjick) [inline web/web/tests/test_graphene.py]: case

**Comment by cballantyne** [comment]
Fixed up casing issues picked up by Erik.

**Pull request merged**